bin/main.rkt

```diff
@@ -519,4 +522,4 @@
                                   (error "Yosys failed."))))
        (output-port))]
 
-     [_ (error "Invalid output format.")])])
+     [_ (error "Invalid output format.")])])
```

Comment:
Please add the newline back to the end of this file -- no reason to delete it!

Reply:
Oops, my bad. There was a merge conflict and I think I messed up when trying to resolve it using the text editor, will fix this